src/app/components/feed/FeedPageContent.js

```diff
@@ -0,0 +1,131 @@
+                description="Text explaining what shared feeds are"
+              />
+            }
+          />
+          <FeedSection
+            title={
+              <FormattedMessage
+                id="feedSection.whatCanBeSharedTitle"
+                defaultMessage="What can be shared?"
+                description="Title of section about what can be shared in the shared feeds"
+              />
+            }
+            content={
+              <FormattedHTMLMessage
+                id="feedSection.whatCanBeSharedContent"
+                defaultMessage="Fully control what data each organization will contribute to the shared feed. Share anonymized request information from your Tipline, or create a shared pool of fact-checks, or choose to share everything with your partner workspaces."
+                description="Text explaining what can be shared in the shared feeds"
+              />
+            }
+          />
+          <FeedSection
+            title={
+              <FormattedMessage
+                id="feedSection.howToGetStartedTitle"
+                defaultMessage="How do you get started?"
+                description="Title of section about how to get started with shared feeds"
+              />
+            }
+            content={
+              <FormattedHTMLMessage
+                id="feedSection.howToGetStartedContent"
+                defaultMessage="Create a new shared feed, and select which data from your workspace you would like to contribute. </br> Then invite one or more other workspaces to participate. All contributing workspaces will share the same data."
+                description="Text explaining how to use shared feeds."
+              />
+            }
+          />
+          <FeedSection
+            title={
+              <FormattedMessage
+                id="feedSection.whatComesNextTitle"
+                defaultMessage="What comes next?"
+                description="Title of section about what is coming next for the shared feeds"
+              />
+            }
+            content={
+              <FormattedHTMLMessage
+                id="feedSection.whatComesNextContent"
+                defaultMessage="With a single view into multiple Tiplines, each workspace accesses a new level of insights including trend narratives across audiences."
+                description="Text about what is coming next for the shared feeds"
+              />
+            }
+          />
+        </div>
+      </div>
+      <Can permissions={permissions} permission="create Feed">
+        <span className={styles.createFeedButton}>
+          <ButtonMain
+            className="projects-list__add-feed"
+            label={
+              <FormattedMessage id="feedComponent.newSharedFeed" defaultMessage="New Shared Feed" description="Button label create new shared feed" />
+            }
+            iconLeft={<AddIcon />}
+            variant="contained"
+            size="small"
```

Comment:
@danielevalverde can you change this to `size="default"` in order to match the figma designs?